test.js

```diff
@@ -1230,3 +1262,61 @@ describe('concurrency', function() {
     expect(ParallelApi.jobs).to.equal(17);
   });
 });
+
+describe('getBabelVersion()', function() {
+  it ('returns the correct version', function() {
+    var expectedVersion = require(path.join(__dirname, 'node_modules/babel-core/package.json')).version;
+    expect(ParallelApi.getBabelVersion()).to.equal(expectedVersion);
+  });
+});
+
+describe('workerpool', function() {
+  var parallelApiPath = require.resolve('./lib/parallel-api');
+
+  var stringToTransform = "const x = 0;";
+
+  var options = {
+    inputSourceMap: false,
+    sourceMap: false,
+    plugins: [
+      'transform-strict-mode',
+      'transform-es2015-block-scoping'
+    ]
+  };
+
+  after(function() {
+    return terminateWorkerPool();
+  });
+
+  it('should limit to one pool per babel version', function() {
+    this.timeout(5*1000);
+    delete require.cache[parallelApiPath];
+    process.env.JOBS = '2';
```

Comment:
in `afterEach` we should restore `JOBS` it what is was before this test rand.

Reply:
good catch - will add that so these tests are not leaking state